### PR TITLE
Feature/refprop paths

### PR DESCRIPTION
**Pull request by jowr**
Explicitly specify a REFPROP library and some convenience functions to handle paths on different platforms.

### CODE REVIEW

File: Web/coolprop/REFPROP.rst [hunk 1]
```diff
@@ -87,7 +87,19 @@ The configuration key for setting the REFPROP path (see :ref:`configuration`) is
     
     In [1]: CP.set_config_string(CP.ALTERNATIVE_REFPROP_PATH, 'c:\\Program Files\\REFPROP\\')
 
-If you do this, internally CoolProp will call the ``SETPATH`` function in REFPROP to tell REFPROP that it should find the ``fluids`` and ``mixtures`` directories within this directory.  If you don't do this, CoolProp will use whatever default logic REFPROP uses to find the fluid files.
+If you do this, internally CoolProp will construct the full path to the ``fluids`` and ``mixtures`` directories and use them to call REFPROP's `SETUP` function when loading fluids.  If you don't do this, CoolProp will use whatever default logic REFPROP uses to find the fluid files.
+
+If you wish to use a certain shared library, for example to try different REFPROP versions, you explicitly define it via ``ALTERNATIVE_REFPROP_LIBRARY_PATH``. This canfiguration variable makes CoolProp ignore the ``ALTERNATIVE_REFPROP_PATH`` when loading the shared library and you might have to provide the full path to your shared library here by doing something like this in python:
```
Comment (ibell) [inline Web/coolprop/REFPROP.rst]: "canfiguration"

**Comment by ibell** [comment]
I think that looks good to me.  The logic seems sound, though I did sleep 3 hours last night, so who knows :)

**Comment by ibell** [comment]
Feel free to squash+merge this guy after you fix the typo

**Comment by ibell** [comment]
Looks like there is a small issue with the join_paths, and it should be
RP_join_paths.  All the buildbot builds are dead.

**Comment by jowr** [comment]
Yep, I saw that too.

**Comment by ibell** [comment]
Can you please fix it?

**Pull request merged**